doc/task-hierarchy.md

```diff
@@ -90,14 +88,14 @@
   - `Reasoning -> Deductive Reasoning`
   - `Reasoning -> Discrete Reasoning`
   - `Reasoning -> Logical Reasoning`
+    - `Reasoning -> Logical Reasoning -> Reasoning with Symbols`: Tasks where symbols represent various things e.g. if X is the number of apples in the fridge today morning and Y is the number remaining after I ate a few apples, X-Y is the number of apples I ate.
```

Comment:
Is the reasoning with symbols always logical reasoning?

Reply:
keeping both